File: RelationExample/RelationExample/BankCustomer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RelationExample
{
    class BankCustomer
    {
        private int customerNumber;

        public int GetCustomerNumber()
        {
            return customerNumber;
        }
        public void SetCustomerNumber(int number)
        {
            customerNumber = number;
        }

        public BankCustomer(int customerNumber)
        {
            this.customerNumber = customerNumber;
        }

        public void FindCheckingAccount(CheckingAccount account)
        {
            Console.WriteLine("The account number of the customer with customer number {0} is: {1}", customerNumber, account.GetAccNumber());
        }
    }
}

File: RelationExampleC++/RelationExampleC++/BankCustomer.cpp
#include "BankCustomer.h"


int BankCustomer::GetCustomerNumber()
{
	return m_customerNumber;
}

void BankCustomer::SetCustomerNumber(int number)
{
	m_customerNumber = number;
}

BankCustomer::BankCustomer(int customerNumber) : m_customerNumber(customerNumber)
{}

void BankCustomer::FindCheckingAccount(CheckingAccount account)
{
	std::cout << "The account number of the customer with customer number " << m_customerNumber << " is: " << account.GetAccNumber() << " \n";
}